plugins/woocommerce/templates/single-product/meta.php

```diff
@@ -12,9 +12,11 @@
  *
  * @see         https://woocommerce.com/document/template-structure/
  * @package     WooCommerce\Templates
- * @version     3.0.0
+ * @version     3.0.1
```

Comment:
Here and elsewhere, is the 'minor' bump a means of satisfying the template version bump check (but without actually bumping it to `9.7.0`)?

Reply:
Oops, I thought the version referred to the file itself, not the plugin version. I will update all occurrences accordingly 👍

Reply:
Both updated ✅